### ISSUE TITLE
parsing cli tables

### ISSUE THREAD

**Issue by slavaGanzin** [abhimanyu003/sttr#31]
Hello, Abhimanyu. Awesome tool.

I used [eat](https://github.com/antonmedv/eat) before and I want to switch to sttr, because of more general use cases covered by it, and because it's a single binary cli. Checks all the boxes.

What I do a lot with eat is parsing standard CLI tables like `ps` or `docker images`:
```
$ ps | eat

[
  {
    "PID": "5424",
    "TTY": "pts/1",
    "TIME": "00:00:00",
    "CMD": "fish"
  },
  {
    "PID": "6183",
    "TTY": "pts/1",
    "TIME": "00:00:00",
    "CMD": "ps"
  },
  {
    "PID": "6184",
    "TTY": "pts/1",
    "TIME": "00:00:00",
    "CMD": "node"
  },
  {
    "PID": "6185",
    "TTY": "pts/1",
    "TIME": "00:00:00",
    "CMD": "xclipm"
  }
]

```

I use this as a preprocessing step before manipulating data with something like `jq` or `fx`

Actually, I implemented this in eat. Code is super easy, but it works.
https://github.com/antonmedv/eat/pull/1/commits/0c2bec801ebf93a77b0fd3a6ffc1392766207d48#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R79-R81

Maybe you could look into it.

**Comment by abhimanyu003** [comment]
Hello @slavaGanzin
Thanks so much for your feedback and suggestion really appricate it. 

I just installed eat.  It seems to be breaking for me on mac 

<img width="1159" alt="image" src="https://user-images.githubusercontent.com/265913/212100717-341e0fe0-bfec-4904-9af6-a4a4683d1b58.png">

I also tried couple of commands

```sh
ll | eat
ls -l | eat
```
Also what will happen if the table contains `dash --- or ===` as row separation.

![image](https://user-images.githubusercontent.com/265913/212100431-934f7b4b-fa16-483e-b39d-583931f953f4.png)

**Comment by slavaGanzin** [comment]
Yeah, right. I was using my own fork. Even if my PR was merged inside eat, the npm package was never really updated.

```bash
$ npm i -g slavaGanzin/eat

$ ps | eat                                                                                                                                          [
  {
    "PID": "58382",
    "TTY": "pts/4",
    "TIME": "00:00:02",
    "CMD": "fish"
  },
  {
    "PID": "62264",
    "TTY": "pts/4",
    "TIME": "00:00:00",
    "CMD": "ps"
  },
  {
    "PID": "62265",
    "TTY": "pts/4",
    "TIME": "00:00:00",
    "CMD": "nodem"
  }
]

```
